webapp-django/crashstats/crashstats/models.py

```diff
@@ -193,6 +194,7 @@ def inner(*args, **kwargs):
         self = args[0]
         url_or_implementation = args[1]
         if isinstance(url_or_implementation, basestring):
+            raise NotImplementedError(url_or_implementation)
             url = url_or_implementation
```

Comment:
I'm guessing that the `raise` line is there for testing things out. Is that right? If not, then setting the `url` here is moot.